integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java

```diff
@@ -447,23 +388,8 @@ private static String createAndVerifyDomainImage() {
 
     // create image with model files
     if (!OKD) {
-      String additionalBuildCommands = WORK_DIR + "/" + COPY_WLS_LOGGING_EXPORTER_FILE_NAME;
-      StringBuffer additionalBuildFilesVarargsBuff = new StringBuffer()
-          .append(WORK_DIR)
-          .append("/")
-          .append(WLS_LOGGING_EXPORTER_YAML_FILE_NAME)
-          .append(",")
-          .append(DOWNLOAD_DIR)
-          .append("/")
-          .append(WLE_DOWNLOAD_FILENAME_DEFAULT)
-          .append(",")
-          .append(DOWNLOAD_DIR)
-          .append("/")
-          .append(SNAKE_DOWNLOADED_FILENAME);
-
       logger.info("Create image with model file and verify");
-      miiImage = createMiiImageAndVerify(WLS_LOGGING_IMAGE_NAME, WLS_LOGGING_MODEL_FILE,
-          MII_BASIC_APP_NAME, additionalBuildCommands, additionalBuildFilesVarargsBuff.toString());
+      miiImage = createMiiImageAndVerify(WLS_LOGGING_IMAGE_NAME, WLS_LOGGING_MODEL_FILE,MII_BASIC_APP_NAME);
```

Comment:
Can we change the Variable name WLS_LOGGING_IMAGE_NAME and WLS_LOGGING_MODEL_FILE to reflect something other than WebLogic Logging to avoid confusion.

Reply:
changed